Gemfile

```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
+
+gem 'hashie'
+gem 'rest_client'
+
+group :test do
+  gem 'rspec'
+  gem 'webmock'
+  gem 'pry'
+end
```

Comment:
true enough. We didn't cut a new version of the gem, needed this to be a master branch change to pull for another project. When we cut it though I'll make sure I do this. Thanks, brah.